libusb/os/emscripten_webusb.cpp

```diff
@@ -839,11 +839,15 @@ int em_handle_transfer_completion(usbi_transfer* itransfer) {
 		: usbi_handle_transfer_completion(itransfer, status);
 }
 
+static const char* em_get_driver_name(void) {
+	return "WebUSB";
+}
+
 }  // namespace
 
 #pragma clang diagnostic ignored "-Wmissing-field-initializers"
 extern "C" const usbi_os_backend usbi_backend = {
-	.name = "Emscripten + WebUSB backend",
+	.name = "Emscripten",
```

Comment:
here too - I don't think this is required, is it?

Reply:
`WebUSB` is the name of the driver for `Emscripten` backend. Now that we have somewhere to put it, I think it is more appropriate to have it in `get_driver_name`.